### PR TITLE
startswith requires a tuple, not a list

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: do_like_javac/tools/wpi.py [hunk 1]
```diff
@@ -15,8 +15,11 @@
 
 argparser = None
 
-banned_options = ["nowarn", "classpath", "processorpath", "processor", "Xmaxerrs", "Xmaxwarns", "proc:none", "release", "XepDisableAllChecks"]
```
Comment (kelloggm) [inline do_like_javac/tools/wpi.py:18]: Why did you change `banned_options` to a tuple, as well? For consistency?

I'd be inclined not to change more than is strictly necessary, since we don't have unit tests (yet?) for this codebase.

**Review by kelloggm — CHANGES_REQUESTED** [review]

File: do_like_javac/tools/wpi.py (hunk 1 above)
Comment (mernst) [reply do_like_javac/tools/wpi.py:18]: Just for consistency, and to prevent the code from accidentally side-effecting it.  (Tuples are immutable and there is no need to change it.)

Let me know whether that is sufficient motivation; if it is not, I will change it back to a list.

Comment (kelloggm) [reply do_like_javac/tools/wpi.py:18]: That's fine, as long as the semantics are the same (I think that they are).
